.gitignore

```diff
@@ -22,9 +22,10 @@ modext.js
 .settings/
 nbproject/
 
-# ignore php and txt files beginning with test in root directory
+# ignore dir, php and txt files beginning with test in root directory
 /test*.php
 /test*.txt
+/test
```

Comment:
I don't necessarily mind this being added to the .gitignore file, but it does not belong in this PR with unrelated changes.

Reply:
BAh. Sorry...snuck in there.